Group.go

```diff
@@ -57,6 +57,25 @@ func (g Group) ListMembers(opts ...ListQueryOption) (Users, error) {
 	return marsh.Users, g.graphClient.makeGETAPICall(resource, compileListQueryOptions(opts), &marsh)
 }
 
+// Get a list of the group's members. A group can have users, devices, organizational contacts, and other groups as members.
+// This operation is transitive and returns a flat list of all nested members.
```

Comment:
Please add a hint here, that currently only `User` instances will be returned. See documentation for function `func (g Group) ListMembers(opts ...ListQueryOption) (Users, error) {`, hence: 
```go
// This method will currently ONLY return User-instances of members
```